.github/workflows/build-check.yml

```diff
@@ -22,6 +22,9 @@ jobs:
         uses: actions/setup-go@v3
         with:
           go-version: ~1.19.2
+      - name: Lint Terraform
```

Comment:
The action only uses [terraform fmt -check -recursive -diff](https://github.com/actionshub/terraform-lint/blob/main/entrypoint.sh#L1) but we already [had that](https://github.com/aws/amazon-cloudwatch-agent-test/blob/main/terraform/Makefile#L4-L5) so its better to not use third party and implement ourselves IMO.

Reply:
Changed to use bash directly instead of action.